lib/index.js

```diff
@@ -224,6 +228,31 @@ async function analyzeProjectDependencies(options, pkgData, pkgFile) {
   return output
 }
 
+/** Get peer dependencies from installed packages */
+function getPeerDependencies(current, options) {
+  const basePath = options.cwd || './'
+  return Object.keys(current).map(pkgName => {
+    const path = basePath + 'node_modules/' + pkgName + '/package.json'
+    try {
+      const pkgData = fs.readFileSync(path)
+      const pkg = jph.parse(pkgData)
+      return vm.getCurrentDependencies(pkg, { dep: 'peer', filter: options.filter, reject: options.reject })
```

Comment:
We should probably pass the all of the users' options and just override `dep: 'peer'`:

```js
return vm.getCurrentDependencies(pkg, { ...options, dep: 'peer' })
```

Reply:
Yes, that sounds good. I've fixed it.